### PR TITLE
Fixes Synthdef documentation, solves #3920

### LINKED ISSUE #3920: SynthDef help: erroneous Argument Rates examples
SC 3.9.3

http://doc.sccode.org/Classes/SynthDef.html#Argument%20Rates

1. SynthDef \trigTest:
rates argument should be 
`[0, 0, 4]`

2. SynthDef \trigTest isn't added

3. SynthDef \trigTest2:
rates argument should be 
`[0, \tr, 4]`

### PR DESCRIPTION
**Pull request by geoffroymontel**
[schelp] fixes SynthDef documentation, makes clearer differences between the ways to write SynthDef arguments

Purpose and Motivation
----------------------

Fixes #3920 and makes more clear that for t_ parameters, there's a difference between setting the synth and setting parameter in Pbind or Pmono

Types of changes
----------------

Documentation

Checklist
---------

- [x] All tests are passing

- [x] If necessary, new tests were created to address changes in PR (and tests are passing)

- [X] Updated documentation, if necessary

- [X] This PR is ready for review

### CODE REVIEW

File: HelpSource/Classes/SynthDef.schelp [hunk 1]
```diff
@@ -229,39 +229,61 @@ x.free;
...
 )
 
+
+// Using the first version
+y = Synth(\trigTest1);
+y.set(\t_trig, 1); // you can do this multiple times
+y.set(\t_trig, 1, \freq, 220); // hear how the freq lags
+y.free;
+
+// Pbind and Pmono know about the t_ notation so you should use the parameter trig
```
Comment (jamshark70) [inline HelpSource/Classes/SynthDef.schelp]: Note that it's different for Pbind with the \set event type.

Pbind with the default \note event type, and Pmono, use an automatically-generated function to look up argument values. This function translates \trig in the event to \t_trig in the synth.

With event type \set, you list the desired arguments explicitly and there is no translation. Possibly there should be, but currently there's not.

```
Pbind(
    \type, \set,
    \id, ...,
    \args, #[t_trig],
    \t_trig, 1,
    \dur, ...
)
```

Comment (geoffroymontel) [reply HelpSource/Classes/SynthDef.schelp]: Thanks ! I did not know that, I will change my PR accordingly

Comment (jamshark70) [reply HelpSource/Classes/SynthDef.schelp]: I'm afraid I have to be a little bit fussy, for the sake of accuracy:

> Pbind and Pmono with the default \note event type

Technically, this should be "Pbind with the default \note event type, and Pmono, ..." -- Pmono uses event types \monoNote, \monoSet and \off, so it isn't exactly right to imply that Pmono uses \note.

Comment (geoffroymontel) [reply HelpSource/Classes/SynthDef.schelp]: I like accuracy, especially in docs !
Thanks a lot for your review, I changed the file accordingly.

**Comment by geoffroymontel** [comment]
@jamshark70 thanks for your feedback. I changed the documentation accordingly. Are you OK with the result ?

**Comment by brianlheim** [comment]
Thanks @geoffroymontel !

**Pull request merged**